search/views.py

```diff
@@ -152,7 +152,7 @@ def search_view_helper(request):
             'tags_mode': sqp.tags_mode_active(),
             'query_time': results.q_time 
         }))
-
+        
```

Comment:
Nit pick: Consider setting your editor to trim all trailing whitespace automatically or use a code formatter